client/components/MarketOrder.jsx

```diff
@@ -1,12 +1,52 @@
 import React from 'react';
+import "../app.css";
 
-const MarketOrder = (props) => {
-	// console.log('Market order props', props)
-	return (
-		<div>
-			<h1> Market Order </h1>
-		</div>
+class MarketOrder extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+
+			total: 0
+		}
+	}
+	
+  onChangeHandler(e) {
+  	var results = eval(this.props.companies[0].currentDay[0].currentPrice * parseFloat(e.target.value));
+		this.setState({
+    	total: results
+    })
+  }
+
+	render() {
+	// console.log('Market order props', this.props.companies[0].currentDay[0].currentPrice)
+		return(
+			<div>
+
+				<div className="mainMenu">
+					<header className="header-class"> Buy {this.props.companies[0].company}</header>
```

Comment:
This use of access state variables works, but try using state deconstruction -- otherwise, you'll get this lint-error: [eslint] Must use destructuring state assignment (react/destructuring-assignment).

Reply:
eg. const { weeklyData, yearlyData, currentPrice } = this.state; 
these are all properties of the state being deconstructed into their own variables.